Assignment/starter_code/js/app.js

```diff
@@ -0,0 +1,51 @@
+// I create two variables: one for the City, the other to reset the input
+var city;
+var resetfield;
+
+
+$(document).ready(function(){
+
+	$('#submit-btn').click(function(submit){
+		// Use preventdefault to cancel the default action of the click
+		submit.preventDefault(); 
+		// My 2 variables are define here
+		var city = $('#city-type').val();
+		var resetfield = $('#city-type').val('');
+
+
+		// I use removeAttr to remove all classes before assigning a new one
+		$('body').removeAttr("class");
+
+		// Start the conditions using if else
+		if (city === 'nyc'|| city==='new york city'|| city==='new york'){
+			$('body').addClass('nyc'); // Add the css style background here
+			resetfield;
```

Comment:
resetfield doesnt do anything here,

infact the only part that works is the ```$('#city-type').val('');``` which resets the field.  You decided to assign it to a variable and call it in here which really doesnt do anything.  It's almost similar to doing

```
var a = 5;
a;
```

Can you see how a is really just a variable not doing anything?